nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPath.java

```diff
@@ -564,34 +571,39 @@ public static boolean isMapPut(Symbol.MethodSymbol symbol, VisitorState state) {
   }
 
   /**
-   * root of an access path; either a variable {@link javax.lang.model.element.Element} or <code>
-   * this</code> (enclosing method receiver)
+   * root of an access path; either an {@link javax.lang.model.element.Element} (representing a
+   * local variable, field, or zero-argument static method call) or {@code this} (enclosing method
+   * receiver)
    */
   public static final class Root {
 
     /** does this represent the receiver? */
     private final boolean isMethodReceiver;
 
-    @Nullable private final Element varElement;
+    /** if this does not represent the receiver, the element for this */
+    @Nullable private final Element element;
 
-    Root(Element varElement) {
+    Root(Element element) {
       this.isMethodReceiver = false;
-      this.varElement = Preconditions.checkNotNull(varElement);
+      this.element = element;
```

Comment:
Runtime check no longer needed because NullAway checks itself?

Reply:
Yes, we probably have other `Preconditions.checkNotNull()` calls that are passed `@NonNull` arguments; probably should add a warning for that at some point